File: 15.js
const readline = require("readline").createInterface({
    input: process.stdin,
    output: process.stdout
});
readline.question("Enter a character: ", input => {
    const x = input.charAt(0);
    if (/[a-zA-Z]/.test(x)) {
        console.log("It is an alphabet");
    } else if (/[0-9]/.test(x)) {
        console.log("It is a digit");
    } else {
        console.log("It is a special character");
    }
    readline.close();
});

File: 15.c
#include <stdio.h>
#include <ctype.h>

int main() 
{
    char x;
    printf("Enter a character: ");
    scanf(" %c", &x); 
    if (isalpha(x)) 
    {
        printf("It is an alphabet\n");
    }
    else if (isdigit(x)) 
    {
        printf("It is a digit\n");
    }
    else 
    {
        printf("It is a special character\n");
    }
    return 0;
}
